document alternative sign in method

gidsg opened the pull request:
Document an alternative sign-in method for Logit, which I have found simpler to that currently documented.

gidsg committed:
document alternative sign in method

richardTowers commented on source/documentation/logging/logit-joiners.md:
```diff
@@ -4,10 +4,10 @@ If you do not have a Logit account you can create one by signing into Logit for
-1. Click on the 9 small dots at the top right of the page.
-1. click **More** at the bottom to show more options.
-1. scroll down and click on the **Logit** icon.
+1. Go to https://logit.io/ and click Sign In
```
You can save a click (at the moment) by going straight to https://logit.io/dashboard.

This will also remember the email you logged in with last (once you've done the other steps once).

richardTowers reviewed (dismissed):
Great find - still annoying that clicking the "Log in with Google" button does the wrong thing, but this is much nicer.

soupdragon99 committed:
Tech writer edits

I made a couple of style changes to match our guidelines. Happy to chat if you need anything else.

soupdragon99 reviewed (dismissed):
Looks great, I just made some small changes to match our style guidelines for this type of content.

gidsg committed:
Use URL suggested by @richardTowers

gidsg commented:
OK, I have made the change suggested by @richardTowers 

Would someone from RE like to merge this PR, or shall I?
